services/backend/src/recommendation_system/recommendation_flow/model_prediction/AlphaModel.py

```diff
@@ -0,0 +1,118 @@
+import random
+# from src import db
+import pandas as pd 
+import numpy as np
+from collections import defaultdict
+# from sqlalchemy.sql import text
+import os
+import pickle
+import sys
+import subprocess
+from .AbstractModel import AbstractModel
+
+
+class AlphaModel(AbstractModel):
+    def predict_probabilities(self, content_ids, user_id, seed=None, **kwargs):
+        
+        try:
+            print('try use tfdf')
+        #     # import tensorflow_decision_forests as tfdf
+            subprocess.check_call([sys.executable, "-m", "pip", "install", "tensorflow_decision_forests"])
+            import tensorflow_decision_forests as tfdf
+
+            with open('/usr/src/app/gbdt_model_v3.pickle', 'rb') as f:
+                GBDT_model = pickle.load(f)
+            with open('/usr/src/app/content_artist_style_dic.pickle', 'rb') as f:
+                dic_id_style = pickle.load(f)
+            with open('/usr/src/app/prediction_prep_dic.pickle', 'rb') as f:
+                prep_dic = pickle.load(f)
+            with open('/usr/src/app/dic_id_to_embedding.pickle', 'rb') as f:
```

Comment:
nit: https://stackoverflow.com/questions/893333/multiple-variables-in-a-with-statement